### DISCUSSION TITLE
Repository Name DakotaPride/Incantation has been retired and cannot be reused

### DISCUSSION THREAD

**Discussion by DakotaPride** [community/community#20910] [Other Feature Feedback, Questions, & Ideas]
Hello!
I found that if I wanted to fix this issue, I should create a support ticket. I apologize if this is not what was meant by 'support ticket'. Recently, I wanted to create a community/organization of sorts but very quickly decided against it. I forgot to transfer back ownership of a repository from the organization to myself and received the error in the title.

Repository That I'm Trying To Transfer:
[HoloScopeStudios/Incantation](https://github.com/HoloScopeStudios/Incantation)

My Userpage (not sure if this is required to know, but saw that it was commonly put in the body of these issues):
[DakotaPride](https://github.com/DakotaPride)

If you need any more information, please let me know so that I may provide it!

**Comment by radar-blip** [comment]
Here is a link to GitHub support (and creating a ticket) if its of any help: 
https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket

Stay tuned for others to chime in. 
Have a great day!

**Reply by DakotaPride** [reply]
This just leads me to the community forums (after selecting the categories my issue falls under). Of which I was told there to create a support ticket as well. This, again, just took me back to the forums. I'm not sure if I'm just doing something wrong or if I have already created a support ticket or something, but yeah.
